js/signin_up/signin.js

```diff
@@ -0,0 +1,91 @@
+import { ERROR_MESSAGE } from "../../constant.js";
+import * as common from "./common_login.js";
+import { emailCheck, passwordCheck, isMatch, checkEmailDupli } from "../validation.js"; 
+import { inputDeleteNode } from "../node.js";
+import { emailDiv, pwdDiv, pwdDiv2, emailInput, pwdInput, signinBtn, pwdEyeIcon, pwdEyeIcon2, pwdInput2 } from "../declaration.js";
+
+let emailVal = "", pwdVal = "", pwdVal2 = "";
+
+// 비밀번호 input 핸들러 함수
+function passwordHandlerFuc(password) {
+  if(password) {
+    passwordCheck(password) ? inputDeleteNode('password') : common.errorMsg("wrongPwd")
+  } else {
+    common.errorMsg("NoPwd");
+  }
+}
+
+// 이메일 input 핸들러
+function emailHandlerFunc(email) {
+  email ? (
+    checkEmailDupli(email) ? 
+      emailCheck(email) ? inputDeleteNode('email') : common.errorMsg("wrongEmail")
+    : common.errorMsg("inUseEmail")
+  ) : common.errorMsg("NoEmail")
+  emailVal = email; 
```

Comment:
요렇게 삼항연산자가 두 개 이상 많아지면 코드 line수는 확실히 줄어들지만, 동시에 여러 경우를 생각해야 해서 살짝 가독성이 떨어질 수 있을 것 같아요.
```suggestion
  if(!email){
    common.errorMsg("NoEmail")
    return;
  }
  if(!checkEmailDupli(email)){
    common.errorMsg("inUseEmail")   
    return;
  }
  emailCheck(email) ? inputDeleteNode('email') : common.errorMsg("wrongEmail")
   
 

```